README.md

```diff
@@ -331,6 +331,8 @@ For more detailed description, go through _kube-fledged's_ [design proposal](doc
 
 `--job-priority-class-name:` priorityClassName of jobs created by kubefledged-controller.
 
+`--job-retention-policy:` Set if an Image Manager Kubernetes Job API object will be deleted or retained (for debugging) after it finishes. Possible values are 'delete' and 'retain', defaults to 'delete'.
```

Comment:
Please rephrase as below:-
`--job-retention-policy:` Determines if the jobs created by kubefledged-controller would be deleted or retained (for debugging) after it finishes. Possible values are 'delete' and 'retain'. default value is 'delete'.